pkg/keto/cmd/create.go

```diff
@@ -75,6 +75,11 @@ func validateCreateFlags(c *cobra.Command, args []string) error {
 }
 
 func createClusterCmdFunc(c *cobra.Command, args []string) error {
+	cli, err := newCLI(c)
+	if err != nil {
+		return err
```

Comment:
I like returning errors instead of logging them, because if you log an error then it is no longer an error. But if you return an error, the caller can decide what to do with it, either log it or return further up the call stack.